docs/docs/supply-chain/sbom.md

```diff
@@ -737,6 +739,31 @@ $ cat result.spdx.json | jq .
 
 </details>
 
+## Scanning
+Trivy can take SBOM documents as input for scanning.
+See [here](../target/sbom.md) for more details.
+
+Also, Trivy searches for SBOM files in container images.
```

Comment:
I think it should be clarified that this currently **only** works for Bitnami images.

I came here from reading the release notes that just state:

```
 9ef0113 feat: add SBOM analyzer (#​4210)
```

This got me excited, because we could use this for our own images, but unfortunately this currently only works for this hardcoded directory.

Reply:
Why do you think it works with the hardcoded directory?

Reply:
I am referring to this discussion: https://github.com/aquasecurity/trivy/pull/4210/files#r1212955004

Even in current `main`, the `sbom.go` only checks the hardcoded path `/opt/bitname` for spdx-SBOMs:

```go
// For Bitnami images
	if strings.HasPrefix(input.FilePath, "opt/bitnami/") {
```

Or am I reading this wrong and other paths/formats are somehow implicitly supported?

Reply:
No, it is just rewriting file paths for Bitnami. SBOM itself is analyzed anyway.